qutebrowser/browser/webkit/webkitsettings.py

```diff
@@ -88,21 +88,9 @@ def _set_user_stylesheet():
     QWebSettings.globalSettings().setUserStyleSheetUrl(url)
 
 
-def _init_private_browsing():
-    if config.get('general', 'private-browsing'):
-        if qtutils.is_qtwebkit_ng():
-            message.warning("Private browsing is not fully implemented by "
-                            "QtWebKit-NG!")
```

Comment:
The next QtWebKit-NG release is going to implement private browsing, and I didn't see an easy way to check if it's implemented or not - so I decided to drop the message.